### ISSUE TITLE
Disabled spaces can be clicked

### ISSUE THREAD

**Issue by JammingBen** [owncloud/web#11334]
## Describe the bug
Disabled spaces can be clicked despite them being unaccessible. The tiles view then shows an error, the table will just do nothing.

## Steps to reproduce
1. Create a new space
2. Disable the space
3. Enable the filter "Include disabled"
4. Click the space

## Expected behavior
The space shouldn't be clickable. Also, the mouse curser should reflect that by *not* showing a pointer.

## Actual behavior
The space is clickable. Also, the mouse curser is a pointer on the element.

**Comment by AlexAndBear** [comment]
Yeah, something changed here for sure, in tiles view should be the disabled text over the icon as well
